src/app/modules/browse/browse-controller.js

```diff
@@ -56,7 +63,13 @@
 		$scope.$watch('bookCountObject.bookCount', function() {
 			$scope.bookCount = $scope.bookCountObject.bookCount;
 		});
-        
+
+        $scope.toggleNext = function(event) {
+            var list = event.currentTarget.nextElementSibling;
+
+            $(list).slideToggle();
```

Comment:
Is this our function or a library we don't control? I don't find it a very obvious name for toggling visibility.